kotlin-eclipse-ui/src/org/jetbrains/kotlin/ui/editors/completion/KotlinCompletionUtils.java

```diff
@@ -52,7 +55,14 @@ public JetSimpleNameExpression getSimpleNameExpression(@NotNull JavaEditor edito
         String sourceCode = EditorUtil.getSourceCode(editor);
         String sourceCodeWithMarker = new StringBuilder(sourceCode).insert(identOffset, KOTLIN_DUMMY_IDENTIFIER).toString();
         
-        JetFile jetFile = KotlinPsiManager.INSTANCE.parseText(StringUtilRt.convertLineSeparators(sourceCodeWithMarker), EditorUtil.getFile(editor));
+        JetFile jetFile = null;
+        IFile file = EditorUtil.getFile(editor);
+        if (file != null) {
+            jetFile = KotlinPsiManager.INSTANCE.parseText(StringUtilRt.convertLineSeparators(sourceCodeWithMarker), file);
+        } else {
+            KotlinLogger.log(IStatus.ERROR, "Failed to retrieve IFile from editor " + editor, null);
```

Comment:
If file is `null` you can return null in this else-block and declare variabe jetFile as `JetFile jetFile;` - without assigning it to `null`. Compiler will check that jetFile was assigned in then-block